cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/StreamSizeScheduler.java

```diff
@@ -493,11 +532,11 @@ public void createScheduleTask(Id.Program programId, SchedulableProgramType prog
                                    StreamSizeSchedule streamSizeSchedule, boolean active,
                                    long basePollSize, long basePollTs, long lastRunSize, long lastRunTs,
                                    boolean persist) throws SchedulerException {
```

Comment:
perhaps you should rename the parameter "persist" to something else. It does not indicate whether a schedule is persisted (all schedules are persisted, right?). It indicates that this is the first time the schedule is created. Or if false, it indicates that we're in the startup sequence and we are reinstating the schedules that were previously persisted.